ydb/core/base/table_vector_index.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+namespace NKikimr {
+namespace NTableIndex {
+namespace NTableVectorKmeansTreeIndex {
```

Comment:
Btw you also can write like this
```suggestion
namespace NKikimr::NTableIndex::NTableVectorKmeansTreeIndex {
```

Reply:
Fixed